api/src/main/kotlin/edu/wgu/osmt/collection/Collection.kt

```diff
@@ -20,11 +21,17 @@ data class Collection(
     val uuid: String,
     val name: String,
     val author: Keyword? = null,
+    val workspaceOwner: String? = null,
+    val status: PublishStatus,
     override val archiveDate: LocalDateTime? = null,
     override val publishDate: LocalDateTime? = null
 ) : DatabaseData, HasUpdateDate, PublishStatusDetails {
 
     fun canonicalUrl(baseUrl: String): String = "$baseUrl/api/collections/${uuid}"
+
+    fun isWorkspace() : Boolean {
+        return  (this.status == PublishStatus.Workspace && StringUtils.isNoneEmpty(this.workspaceOwner))
```

Comment:
this.workspaceOwner is a String. StringUtils.isNoneEmpty is meant to check list elements. Is this intended?

Reply:
oops, good catch, no I meant to use isNot, which should check for null or blank string. Correcting...